uportal-war/src/main/webapp/WEB-INF/flows/edit-account/editLocalAccount.jsp

```diff
@@ -62,62 +62,78 @@
+                                    <td class="attribute-name">
+                                        <label for="${n}userUsername">
+                                            <spring:message code="username" />
+                                        </label>
+                                    </td>
+                                    <td>
+                                        <form:input path="username" id="${n}userUsername" class="form-control"/>
+                                    </td>
                                 </tr>
                             </c:if>
                             <!--  Password and confirm password -->
                             <tr>
-                                <td class="attribute-name"><spring:message code="password"/></td>
-                                <td><form:password path="password"/></td>
+                                <td class="attribute-name">
+                                    <label for="${n}userPassword">
+                                        <spring:message code="password"/>
+                                    </label>
+                                </td>
+                                <td>
+                                    <form:password path="password" id="${n}userPassword" class="form-control"/>
+                                </td>
                             </tr>
                             <tr>
-                                <td class="attribute-name"><spring:message code="confirm.password"/></td>
-                                <td><form:password path="confirmPassword"/></td>
+                                <td class="attribute-name">
+                                    <label for="${n}userConfirmPassword">
+                                        <spring:message code="confirm.password"/>
+                                    </label>
+                                </td>
+                                <td>
+                                    <form:password path="confirmPassword" id="${n}userConfirmPassword" class="form-control"/>
+                                </td>
                             </tr>
-
                         </tbody>
                     </table>
 
                 </div>
             </div>
-        
+
             <!-- Portlet Section -->
             <div class="portlet-section" role="region">
                 <div class="titlebar">
                     <h3 class="title" role="heading"><spring:message code="standard.attributes"/></h3>
                 </div>
                 <div id="${n}standardAttributes" class="content">
-                
-                    <table class="portlet-table table table-hover">
-                        <thead>
-                            <tr>
-                                <th><spring:message code="attribute.name"/></th>
-                                <th><spring:message code="attribute.value"/></th>
-                            </tr>
-                        </thead>
```

Comment:
yes because this table is a layout table
If the table is used only to visually layout elements within the content
Check that the table does not contain any th elements.
Check that the table element does not contain a non-empty summary attribute.
Check that the table element does not contain a caption element.
[F46: Failure of Success Criterion 1.3.1 due to using th elements, caption elements, or non-empty summary attributes in layout tables](https://www.w3.org/TR/WCAG-TECHS/F46.html)